frontend/src/metabase/query_builder/components/view/NativeQueryPreviewSidebar/NativeQueryPreviewSidebar.tsx

```diff
@@ -107,7 +99,6 @@ export const NativeQueryPreviewSidebar = (): JSX.Element => {
               fontSize={12}
               style={{ backgroundColor: color("bg-light") }}
               showPrintMargin={false}
-              wrapEnabled={true}
```

Comment:
This was a specific feedback from the design team to remove line wrapping and to allow horizontal scrolling instead.

This change will make even more sense when we make the sidebar resizable.